flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/sink/writer/AsyncSinkWriter.java

```diff
@@ -71,10 +71,10 @@
     private final SinkWriterMetricGroup metrics;
 
     /* Counter for number of bytes this sink has attempted to send to the destination. */
-    private final Counter numBytesSendCounter;
+    private final Counter numBytesOutCounter;
 
     /* Counter for number of records this sink has attempted to send to the destination. */
-    private final Counter numRecordsSendCounter;
```

Comment:
The decision is to point both `numXXXSendCounter` and `numXXXOutCounter` to the same number. Removing `numXXXSendCounter` breaks the backward compatibility with 1.15.x